packages/frontend-shared/src/locales/i18n.ts

```diff
@@ -19,6 +19,7 @@ export const VueI18n = createI18n()
 export function createI18n (opts = {}) {
   return _createI18n<MessageSchema, 'en-US'>({
     locale: 'en-US',
+    // @ts-expect-error
```

Comment:
has to do with the key for the message value pair can be any string, where as the type is expecting `en-us`

Reply:
oof, this is a huge shortcoming of that library when it comes to typescript... how about we coerce the type instead? e.g.

```typescript
    /**
     * precompiled messages from unplugin-vue-i18n do not include explicit keys derived from
     * the filenames of the raw message sources, so it must be coerced.
     */
    messages: compiledMessages as { 'en-US': MessageSchema },
 ```